TrustKit/TrustKit.m

```diff
@@ -197,7 +199,14 @@ static void initializeTrustKit(NSDictionary *trustKitConfig)
         // or the swizzling logic when calling [TrustKit resetConfiguration]
         dispatch_once(&dispatchOnceTrustKitInit, ^{
             // Create our reporter for sending pin validation failures; do this before hooking NSURLSession so we don't hook ourselves
-            _pinFailureReporter = [[TSKBackgroundReporter alloc]initAndRateLimitReports:YES];
+            BOOL shouldRateLimitReports = YES;
+            
+            if ([[_trustKitGlobalConfiguration allKeys] containsObject: kTSKShouldRateLimitReports])
```

Comment:
The value is an `NSNumber`, meaning it can be tested for nil independent of truthiness.  It is not necessary to pull the keys into a set and test for inclusion. Try this:

```
NSNumber *rateLimitOverride = _trustKitGlobalConfiguration[kTSKShouldRateLimitReports];
BOOL shouldRateLimitReports = rateLimitOverride ? rateLimitOverride.boolValue : YES;
```